addons/interaction/XEH_postInit.sqf

```diff
@@ -108,3 +108,11 @@ GVAR(isOpeningDoor) = false;
 
 ["isNotSwimming", {!underwater (_this select 0)}] call EFUNC(common,addCanInteractWithCondition);
 ["isNotOnLadder", {getNumber (configFile >> "CfgMovesMaleSdr" >> "States" >> animationState (_this select 0) >> "ACE_isLadder") != 1}] call EFUNC(common,addCanInteractWithCondition);
+
+["ace_settingsInitialized", {
+    if (GVAR(disableNegativeRating)) then {
+        ACE_player addEventHandler ["HandleRating", {
```

Comment:
No, @SilentSpike is right. The handler gets assigned to the unit that is `ACE_player` at the time the settings get initialized. Therefore it's not safe to use `ACE_player` because there is a slight risk that this EH could get assigned to a remote unit.
Btw I also changed this to `player` already